src/extensions/exception-autocapture/error-conversion.ts

```diff
@@ -11,7 +11,42 @@ import {
 import { defaultStackParser, StackFrame } from './stack-trace'
 
 import { isEmptyString, isNumber, isString, isUndefined } from '../../utils/type-utils'
-import { ErrorEventArgs, ErrorProperties, SeverityLevel, severityLevels } from '../../types'
+import { ErrorEventArgs, SeverityLevel, severityLevels } from '../../types'
+
+export interface ErrorProperties {
+    $exception_type: string
+    $exception_message: string
+    $exception_level: SeverityLevel
+    $exception_list?: Exception[]
+    // $exception_source?: string
+    // $exception_lineno?: number
+    // $exception_colno?: number
+    $exception_DOMException_code?: string
+    $exception_is_synthetic?: boolean
+    // $exception_stack_trace_raw?: string
+    $exception_handled?: boolean
+    $exception_personURL?: string
+    $exception_language?: string
+}
```

Comment:
One for the future but would be cool to have a shared schema similar to what we have for TS <-> Python we have for queries

Reply:
will rm the commented out stuff 👀